### PR TITLE
new file:   packages/cpm/cpm.10.2.1/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
New functionalities in the Utls module; which is no more private.

Utls.{lines_of_file|train_test_split|shuffle_then_cut|list_nparts|cv_folds|shuffle_then_nfolds}

### CODE REVIEW

**Comment by mseri** [comment]
Thanks, the failures seem to be unrelated
